Mailer/app/api/v1/Handlers/Users/UserPreferencesHandler.php

```diff
@@ -59,23 +60,20 @@ public function handle($payload)
         foreach ($rows as $row) {
             $variants = [];
             foreach ($subscribedVariants[$row->id] ?? [] as $variant) {
-
                 $variants[] = [
                     'id' => $variant->mail_type_variant_id,
                     'code' => $variant->code,
                     'title' => $variant->title,
                 ];
             }
 
-            $mailType =  $row->ref('mail_types', 'mail_type_id');
-
             $output[$row->mail_type_id] = [
                 'id' => $row->mail_type_id,
-                'code' => $mailType->code,
-                'title' => $mailType->title,
+                'code' => $row->mail_type->code,
+                'title' => $row->mail_type->title,
                 'is_subscribed' => $row->subscribed,
                 'variants' => $variants,
-                'updated_at' => $row->updated_at,
+                'updated_at' => DateTime::from($row->updated_at)->format(DateTime::RFC3339),
```

Comment:
As with the other date-related comment, any date fetched from the DB is already a `DateTime`. I stumbled upon this accidentally and I probably didn't see all instances of this usage. Please review the whole PR for such usage.